app/views/directives/osc-persistent-volume-claim.html

```diff
@@ -2,37 +2,44 @@
   <fieldset ng-disabled="claimDisabled">
     <div ng-show="storageClasses | hashSize" class="form-group">
       <!--storage class-->
-      <label class="required">Storage Classes</label>
-      <div id="claim-storage-class-help" class="help-block mar-bottom-lg">
+      <label>Storage Class</label>
+      <div>
+      <ui-select
+        ng-if="storageClasses"
+        ng-model="claim.storageClass"
+        theme="bootstrap"
+        search-enabled="true"
+        title="Select a storage class"
+        class="select-role"
+        flex>
```

Comment:
Missed this on previous review, but I don't think `flex` is necessary here. Can you try without?